### PR TITLE
Added possibility to define extra Init Containers

### PR DESCRIPTION
**Pull request by davidseidel**
In order to allow extra initialization procedures (e.g. load additional artifacts), I have added the possibility to define extra Init Containers.

### CODE REVIEW

File: charts/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube is an open sourced code quality scanning tool
-version: 9.2.7
+version: 9.2.8
```
Comment (rjkernick) [inline charts/sonarqube/Chart.yaml:4]: Please bump the minor version instead as this adds a new feature

File: charts/sonarqube/README.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `initContainers.image`                | Change init container image                                                                                                 | `busybox:1.32`                                   | 
 | `initContainers.securityContext`      | SecurityContext for init containers                                                                                         | `nil`                                            | 
 | `initContainers.resources`            | Resources for init containers                                                                                               | `{}`                                             | 
+| `extraInitContainers`                 | Extra init containers to e.g. download required artifacts                                                                   | `{}`                                             |
```
Comment (rjkernick) [inline charts/sonarqube/README.md:100]: Please add a section to the values.yaml as well.

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: charts/sonarqube/README.md (hunk 2 above)
Comment (davidseidel) [reply charts/sonarqube/README.md:100]: done

File: charts/sonarqube/Chart.yaml (hunk 1 above)
Comment (davidseidel) [reply charts/sonarqube/Chart.yaml]: done

**Review by rjkernick — APPROVED** [review]

**Comment by davidseidel** [comment]
Thank you @rjkernick for accepting my changes.

**Pull request merged**